doc/reference-py-sack.rst

```diff
@@ -101,6 +101,14 @@
     Enable the repository identified by a string *name*. Packages in that
     repository can be fetched by Queries or Selectors.
 
```

Comment:
Could you please also add here something like:
`  .. see https://github.com/rpm-software-management/hawkey/pull/87 for the details`
so that we will remember the rationale?